Example/Tests/GINIMultipageReviewControllerTests.swift

```diff
@@ -0,0 +1,187 @@
+                       "Fifth toolbar item should be the rotateButton")
+    }
+    
+    func testToolBarAndBottomCollectionContainerColors() {
+        XCTAssertEqual(vc.toolBar.barTintColor, vc.bottomCollectionContainer.backgroundColor,
+                       "toolbar and bottom collection container background colors should match")
+    }
+    
+    func testDatasourceOnDelete() {
+        let vc = MultipageReviewController(imageDocuments: imageDocuments)
+        _ = vc.view
+        vc.view.setNeedsLayout()
+        vc.view.layoutIfNeeded()
+        
+        (vc.deleteButton.customView as? UIButton)?.sendActions(for: .touchUpInside)
+        
+        XCTAssertEqual(vc.mainCollection.numberOfItems(inSection: 0), 2,
+                       "main collection items count should be 2")
+        XCTAssertEqual(vc.bottomCollection.numberOfItems(inSection: 0), 2,
+                       "bottom collection items count should be 2")
+    }
+    
+    func testCellReloadedOnReordering() {
+        let expect = expectation(description: "Delayed reloading on reordering has finished")
+        let currentIndexPath = IndexPath(row: 0, section: 0)
+        let destinationIndexPath = IndexPath(row: 2, section: 0)
+        var updatedImageDocument: [GiniImageDocument] = []
+        
+        vc.didUpdateDocuments = { updatedDocuments in
+            updatedImageDocument = updatedDocuments
+            expect.fulfill()
+        }
+        
+        vc.collectionView(vc.bottomCollection, moveItemAt: currentIndexPath, to: destinationIndexPath)
+        
+        wait(for: [expect], timeout: 1)
+        let firstCell = vc.collectionView(vc.bottomCollection,
+                                          cellForItemAt: IndexPath(row: 0, section: 0)) as? MultipageReviewBottomCollectionCell
+        let secondCell = vc.collectionView(vc.bottomCollection,
+                                           cellForItemAt: IndexPath(row: 1, section: 0)) as? MultipageReviewBottomCollectionCell
+        let thirdCell = vc.collectionView(vc.bottomCollection,
+                                          cellForItemAt: IndexPath(row: 2, section: 0)) as? MultipageReviewBottomCollectionCell
+        
+        XCTAssertEqual(firstCell?.documentImage.image, updatedImageDocument[0].previewImage,
+                       "Second cell image should match the one passed in the initializer")
+        XCTAssertEqual(firstCell?.pageIndicator.text, "1",
+                       "First cell indicator should match its position")
+        XCTAssertEqual(secondCell?.documentImage.image, updatedImageDocument[1].previewImage,
+                       "Second cell image should match the one passed in the initializer")
+        XCTAssertEqual(secondCell?.pageIndicator.text, "2",
+                       "Second cell indicator should match its position")
+        XCTAssertEqual(thirdCell?.documentImage.image, updatedImageDocument[2].previewImage,
```

Comment:
Can't you just verify the switch in one assert?

Reply:
I want to check that the other cells also have the expected value, that is why I check every label an image.